src/main/java/org/jitsi/videobridge/rest/PublicClearPortRedirectBundleActivator.java

```diff
@@ -0,0 +1,173 @@
+/*
+ * Copyright @ 2015 Atlassian Pty Ltd
```

Comment:
2018

Reply:
eee copy paste should automatically replace this :)